beast-gtk/bsttrackrollctrl.cc

```diff
@@ -565,7 +565,8 @@ pointer_move (BstTrackRollController *self,
       drag->type != GXK_DRAG_DONE) /* skip release events */
     {
       guint tick = bst_track_roll_controller_quantize (self, drag->current_tick);
-      bse_proxy_set (self->song, "tick-pointer", tick, NULL);
+      Bse::SongH song = Bse::SongH::down_cast (bse_server.from_proxy (self->song));
```

Comment:
It might (soon) be time to convert self->song into a scoped handle. Please keep an eye on when it's easier or not too much effort to use a Bse::SongS in the track controller and go for it, so we can avoid all those from_proxy down casts.